handler.go

```diff
@@ -47,10 +47,10 @@ func (a *App) getPerson(w http.ResponseWriter, r *http.Request) {
 	if err := p.getPerson(a.DB, id); err != nil {
 		switch err.Error() {
 		case "no rows in result set":
```

Comment:
If you avoid the string match, maybe this can be faster. 

```suggestion
	if err := p.getPerson(a.DB, id); err != nil {
		switch err {
		case pgx.ErrNoRows:
```

Reply:
Could even use some newer error handling functions:

```go
	if err := p.getPerson(a.DB, id); err != nil {
		if errors.Is(err, pgx.ErrNoRows) {
		   respondWithError(w, http.StatusNotFound)
		   return
		}
		respondWithError(w, http.StatusInternalServerError)
		return
    }		
```